web/modules/custom/feedback/src/Form/FeedbackForm.php

```diff
@@ -0,0 +1,227 @@
+    );
+    // Add email field.
+    $form['email'] = array(
+      '#title' => $this->t('Your Email'),
+      '#type' => 'email',
+      '#size' => 25,
+      '#required' => TRUE,
+      // If form editing then insert default values from database to this input.
+      '#default_value' => (isset($record['email']) && $getFilterValue) ? $record['email'] : '',
+    );
+    // Add textarea field.
+    $form['message'] = array(
+      '#title' => $this->t('Your Message'),
+      '#type' => 'textarea',
+      '#required' => TRUE,
+      // If form editing then insert default values from database to this input.
+      '#default_value' => (isset($record['message']) && $getFilterValue) ? $record['message'] : '',
+    );
+    $form['actions']['#type'] = 'actions';
+    // Submit button.
+    $form['actions']['submit'] = array(
+      '#type' => 'submit',
+      '#value' => $this->t('Send'),
+    );
+    return $form;
+  }
+
+  /**
+   * Start validation form.
+   *
+   * {@inheritdoc}.
+   */
+  public function validateForm(array &$form, FormStateInterface $form_state) {
+
+    // Check length name.
+    if (strlen($form_state->getValue('name')) < 5) {
+      $form_state->setErrorByName('name', $this->t('Name is too short.'));
+    }
+    // Check length message.
+    if (strlen($form_state->getValue('message')) < 5) {
+      $form_state->setErrorByName('message', $this->t('Message is too short.'));
+    }
+
+    // Get configuration form value.
+    $config = $this->configFactory->getEditable('feedback.settings');
+    $configuration = $config->get('allowed_value');
+
+    // Get user's email.
+    $email_now = $form_state->getValue('email');
+
+    // Check email in the database.
+    $query = $this->database->select('feedback', 'f');
+    $query->addField('f', 'email');
+    $email_db = $query->execute()->fetchCol();
+
+    // One email to one feedback.
+    // There is a configuration form where you can cancel it.
+    if ($email_db[0] == $email_now && $configuration == 1) {
+      $form_state->setErrorByName('email', $this->t('The email can not be use for form resubmit, because it was already used for sending'));
+    }
+  }
+
+  /**
+   * Start submit form.
+   *
+   * {@inheritDoc}
+   */
+  public function submitForm(array &$form, FormStateInterface $form_state) {
+
+    // Get Feedback id if form edit.
+    $feedback_id = $form_state->getValue('fid');
+    // If we edit feedback, then values update,
+    // else values insert.
+    if ($feedback_id > 0) {
+      // Database connection.
+      $query = $this->database;
+      // Current user.
+      $userId = $this->currentUser->id();
+      // Update feedback.
+      $query->update('feedback')
+        ->fields([
+          'name' => $form_state->getValue('name'),
+          'email' => $form_state->getValue('email'),
+          'message' => $form_state->getValue('message'),
+          'uid' => $userId,
+          'created' => time(),
```

Comment:
it works but in drupal current timestamp is getting using another way - change this